lib/src/setupTests.js

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+const nodeCrypto = require('crypto');
+
+// typescript doesn't like this which is why it's in a JS file
+window.crypto = {
+  getRandomValues: function(buffer) { return nodeCrypto.randomFillSync(buffer); }
```

Comment:
why not just put this in setupTests.ts or move everything frome setupTests.ts to setupTests.js?

Reply:
> why not just put this in setupTests.ts

See the code comment.  Typescript claims that `window.crypto` is readonly (which it is in a real browser) and doesn't let me override it.

> or move everything frome setupTests.ts to setupTests.js?

At least some of the lines in `setupTests.ts` are necessary to get typescript to understand that types for the test functions (for instance that the jest-enzyme matchers exist).

Reply:
Could we just ts-ignore the window.crypto error instead?